### ISSUE TITLE
temperature field beam cross section

### ISSUE THREAD

**Issue by hbc2018fire** [OpenSeesforFire/openseesforfire.github.io#22]
Hi,

is it possible to define a temperature field which depends on x and y within a rectilinear cross section, like eleLoad -ele 2 -beamThermal -source data.dat -0.25 0.25 -0.15 0.15 for an I-section?

**Comment by Tenghankai** [comment]
I have the same question.

**Comment by li-ming-jiang** [comment]
Yes, the temperature field has been enabled. We will update it with manual soon
